lib/addons/src/index.js

```diff
@@ -2,7 +2,7 @@ export class AddonStore {
   constructor() {
     this.loaders = {};
     this.panels = {};
-    this.channel = null;
+    this.channel = { on() {}, emit() {} };
```

Comment:
If we want to have a way to "ignore" the error, maybe that's a feature worth adding, if it's sometimes impossible to avoid the multi-loading problem. Something like `initStoryshots({ nullChannel: true})`?